spec/support/omniauth_helper.rb

```diff
@@ -0,0 +1,12 @@
+def set_invalid_omniauth
+  OmniAuth.config.mock_auth[:facebook] = :invalid_credentials
+end
+OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+  provider: "facebook",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.